src/mapengine/worldwind/manifest.js

```diff
@@ -28,14 +35,50 @@ function initializeWorldwind(args) {
       wwCanvas.style.width = "100%";
       container.appendChild(wwCanvas);
 
+      // TODO pass in an elevationModel that uses local data
       var wwd = new WorldWind.WorldWindow(wwCanvas.id);
 
       // Tell the World Window that we want deep picking.
       wwd.deepPicking = true;
 
       // Add some image layers to the World Window"s globe.
-      wwd.addLayer(new WorldWind.BMNGLayer()); // wms
-      //wwd.addLayer(new WorldWind.BMNGOneImageLayer());
+      var numLayers = args.engine.properties.defaultLayers.length;
+      if (!numLayers) {
+        // TODO uses local data to make a base layer instead of BMNGLayer(reaches back to NASA)
+        wwd.addLayer(new WorldWind.BMNGLayer());
```

Comment:
This will be replaced when the local data solution is set. It is being left for now until a solution is implemented.